pydis_site/apps/resources/views/resources_list.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+from typing import Any, Dict
+
+import yaml
+from django.conf import settings
+from django.http import Http404
+from django.views.generic import TemplateView
+
+RESOURCES_PATH = Path(settings.BASE_DIR, "pydis_site", "apps", "resources", "resources")
+
+
+class ResourcesList(TemplateView):
+    """Shows specific resources list."""
+
+    template_name = "resources/resources_list.html"
+
+    def get_context_data(self, **kwargs) -> Dict[str, Any]:
+        """Add resources and subcategories data into context."""
+        context = super().get_context_data(**kwargs)
+
+        resource_path = RESOURCES_PATH / self.kwargs["type"]
+        if (
+                not resource_path.exists()
+                or not resource_path.is_dir()
+                or not resource_path.joinpath("_category_info.yaml").exists()
+        ):
```

Comment:
`is_dir` returns `False` if the path does not exist

Reply:
I think everything is correct here?

Reply:
yesyes, `not resource_path.exists()` is redundant though